pkg/vsphere/adapter_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2020 VMware, Inc.
```

Comment:
You apparently need to teach that fancy new editor some tricks 😉   ☝️

Reply:
@vaikas did you dump emacs?

Reply:
he's trying goland

Reply:
nice - I used intellij/goland/rubymine/android studio in the past - just learn some keyboard shortcuts and start refactoring

Reply:
Wait, what's wrong here? 🤔

Reply:
There were gofmt/goimport errors here before.